test/dotcom_web/controllers/schedule/predictions_test.exs

```diff
@@ -2,20 +2,11 @@ defmodule DotcomWeb.ScheduleController.PredictionsTest do
   use DotcomWeb.ConnCase, async: true
 
   import DotcomWeb.ScheduleController.Predictions
+  import Mox
```

Comment:
I like to make sure there are empty lines between the different kinds of includes.